git.sh

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Did you mean to check this file in? (Looks like it's your local script for testing??)